evaluator/evaluate.go

```diff
@@ -308,11 +306,12 @@ func loadPackages(tx *gorm.DB, accountID, systemID int,
 	if err != nil {
 		return nil, errors.Wrap(err, "loading packages")
 	}
+
 	pkgByNevra := map[utils.Nevra]namedPackage{}
 	for _, p := range packages {
-		nevra, err := utils.ParseNevra(p.NEVRA)
+		nevra, err := utils.ParseNameEVRA(p.Name, p.EVRA)
 		if err != nil {
-			utils.Log("err", err.Error(), "nevra", p.NEVRA).Warn("Unable to parse nevra")
+			utils.Log("err", err.Error(), "name", p.Name, p.EVRA).Warn("Unable to parse nevra")
```

Comment:
There is odd count of Log() args. It have to be even (name - value pairs) number.

Reply:
Thanks, fixed